### PR TITLE
feat(crons): Add note on consecutive failure alerts

### PR DESCRIPTION
**Pull request by evanpurkhiser**

### CODE REVIEW

File: src/docs/product/crons/alerts.mdx [hunk 1]
```diff
@@ -11,6 +11,14 @@ When your recurring job fails to check-in on schedule or runs beyond its configu
 
 Navigate to the alerts tab in Sentry and create a new issue alert. You should see the issue alert builder. From here, you can start configuring your cron monitor alert rule. First, select the project corresponding to your cron monitor. Then, in the `IF` section of the alert rule configuration, add a new filter. You'll want to select the filter labeled "The event's tags match {key} {match} {value}".
 
+<Note>
+
+You can use the "Number of events in an issue is" trigger filter to only alert you when the monitor has failed multiple consecutive check-ins.
+
+For example if you're job should run every minute you could use the trigger filter **Number of events in an issue is more than 10 in 15 minutes** to be alerted after it's failed 10 times in that window.
```
Comment (davidenwang) [inline src/docs/product/crons/alerts.mdx:18]: your*

**Review by davidenwang — APPROVED** [review]

File: src/docs/product/crons/alerts.mdx [hunk 2]
```diff
@@ -11,6 +11,14 @@ When your recurring job fails to check-in on schedule or runs beyond its configu
...
+
```
Comment (lizokm) [inline src/docs/product/crons/alerts.mdx:19]:
```suggestion

You can use the "Number of events in an issue is" trigger filter to only alert you if the monitor has failed multiple consecutive check-ins. For example, if your job is supposed to run every minute, you could use the trigger filter **Number of events in an issue is more than 10 in 15 minutes**, to be alerted only after it's failed 10 times in that timeframe.

```
Per this rule, shouldn't it be if it's failed MORE than 10 times in that timeframe?? :)

**Review by lizokm — APPROVED** [review]
Made a few small tweaks to make it flow a bit more smoothly.

File: src/docs/product/crons/alerts.mdx (hunk 2 above)
Comment (evanpurkhiser) [reply src/docs/product/crons/alerts.mdx:19]: you're right yeah more

File: src/docs/product/crons/alerts.mdx [hunk 3]
```diff
@@ -11,6 +11,12 @@ When your recurring job fails to check-in on schedule or runs beyond its configu
 
 Navigate to the alerts tab in Sentry and create a new issue alert. You should see the issue alert builder. From here, you can start configuring your cron monitor alert rule. First, select the project corresponding to your cron monitor. Then, in the `IF` section of the alert rule configuration, add a new filter. You'll want to select the filter labeled "The event's tags match {key} {match} {value}".
 
+<Note>
+
+You can use the "Number of events in an issue is" trigger filter to only alert you if the monitor has failed multiple consecutive check-ins. For example, if your job is supposed to run every minute, you could use the trigger filter **Number of events in an issue is more than 10 in 15 minutes**, to be alerted only after it's failed 10 times in that timeframe.
```
Comment (evanpurkhiser) [inline src/docs/product/crons/alerts.mdx:16]:
```suggestion
You can use the "Number of events in an issue is" trigger filter to only alert you if the monitor has failed multiple consecutive check-ins. For example, if your job is supposed to run every minute, you could use the trigger filter **Number of events in an issue is more than 10 in 15 minutes**, to be alerted only after it's failed more than 10 times in that timeframe.
```

Comment (evanpurkhiser) [reply src/docs/product/crons/alerts.mdx:16]: @lizokm

**Pull request merged**